### DISCUSSION TITLE
Add image in Image Field with PDF Forms

### DISCUSSION THREAD

**Discussion by wanaideed** [py-pdf/pypdf#2710] [Q&A]
## Explanation

 I got PDF Forms with Text field and Image Field. How to I add image from Image field?

For text field in document pypdf that show great information and I success. But I fails to add image in Image Field. Any one can help me.

**Comment by pubpub-zz** [comment]
PDF Forms includes Buttons(radio,...)/Text/Choices(Combo lists) and signatures. No image fields I'm aware of can you please provide an example of such a document
